packages/angular_devkit/schematics/BUILD.bazel

```diff
@@ -39,11 +39,9 @@ ts_project(
     name = "schematics_test_lib",
     testonly = True,
     srcs = glob(["src/**/*_spec.ts"]),
-    # TODO: Fix strict_deps failure
-    ignore_strict_deps = True,
     deps = [
         ":node_modules/@angular-devkit/core",
-        ":schematics",
+        ":schematics_rjs",
```

Comment:
Sorry for the unrelated drive by comment, but just curious what does `rjs` stand for?

Reply:
the `_rjs` suffixed package exists as part of our temporary interop as we migrate from one bazel build toolchain to another (from rules_nodejs to rules_js).  It is the actual `ts_project` rule target we ultimately are migrating to.

You can see its definition in [tools/interop.bzl](https://github.com/angular/angular-cli/blob/68ef24d3d7f56dfd8f491b69dc7d882f2b9c860b/tools/interop.bzl)

Reply:
I see, thanks!